frontend/scripts/linux_distribution/appimage/AppImageBuilder.yml

```diff
@@ -19,47 +19,55 @@ AppDir:
     exec_args: $@
   apt:
     arch:
-    - amd64
+      - amd64
```

Comment:
Please update the Flatpak file as well.

https://github.com/flathub/io.appflowy.AppFlowy